docs/source/examples/substrafl/go_further/run_mnist_cyclic.ipynb

```diff
@@ -0,0 +1,1113 @@
+{
```

Comment:
<p>Link generation problem here too, for <code>TorchAlgorithm</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>